modules/mcap/1.4.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+""" Builds mcap.
+"""
+
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "mcap",
+    hdrs = glob(
+        [
+            "include/**/*.inl",
+            "include/**/*.hpp",
+        ],
+        exclude = [
+            "include/mcap/reader.inl",
+            "include/mcap/writer.inl",
+        ],
+    ),
+    includes = ["include"],
+    visibility = ["//visibility:public"],
+    features = [
+        "parse_headers",
+    ],
+    textual_hdrs = [
+        "include/mcap/reader.inl",
+        "include/mcap/writer.inl",
+    ],
+    copts = ["-std=c++17"],
```

Comment:
Could you add a comment explaining that this is only for `parse_headers`?

Reply:
added https://github.com/bazelbuild/bazel-central-registry/pull/2665/commits/2992dee0e03efc2d759a9a51371029b78408b1af